rpc/metrics.go

```diff
@@ -42,9 +42,12 @@ func updateServeTimeHistogram(method string, success bool, elapsed time.Duration
 	}
 	h := fmt.Sprintf("%s/%s/%s", serveTimeHistName, method, note)
 	sampler := func() metrics.Sample {
-		return metrics.ResettingSample(
-			metrics.NewExpDecaySample(1028, 0.015),
-		)
+		var sample metrics.Sample
+		sample = metrics.NewExpDecaySample(1028, 0.015)
+		if !metrics.DisabledRPCHistogramReset {
+			sample = metrics.ResettingSample(sample)
```

Comment:
Could we still reset the timer histogram part, but not reset the count?

Reply:
The main problem is that it doesn't self-decay.  This looks like a much better approach
https://pkg.go.dev/github.com/facebookgo/metrics